core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -80,7 +80,7 @@
  *   {@code org.ehcache}.
  * </P>
  */
-public class EhcacheManager implements PersistentCacheManager, InternalCacheManager {
+public class EhcacheManager implements PersistentCacheManager, InternalCacheManager, HumanReadable {
```

Comment:
Is it desired at this level? I feel the gain from implementing the interface here is really minimal.